### PR TITLE
update examples

### PR DESCRIPTION
**Pull request by Cjkkkk**
1 add DoReFaQuantizer_torch_mnist example.
2 support cuda and cpu for examples.

### CODE REVIEW

File: examples/model_compress/DoReFaQuantizer_torch_mnist.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        batch_size=64, shuffle=True)
+    test_loader = torch.utils.data.DataLoader(
+        datasets.MNIST('data', train=False, transform=trans),
+        batch_size=1000, shuffle=True)
+
+    model = Mnist()
+    model = model.to(device)
+    '''you can change this to DoReFaQuantizer to implement it
+    DoReFaQuantizer(configure_list).compress(model)
+    '''
```
Comment (chicm-ms) [inline examples/model_compress/DoReFaQuantizer_torch_mnist.py]: is it already DoReFaQuantizer ? this comments seems useless.

File: examples/model_compress/MeanActivation_torch_cifar10.py [hunk 2]
```diff
@@ -82,7 +83,8 @@ def main():
 
     # Test base model accuracy
     print('=' * 10 + 'Test on the original model' + '=' * 10)
-    model.load_state_dict(torch.load('vgg16_cifar10.pth'))
+    if os.path.isfile('vgg16_cifar10.pth'):
+        model.load_state_dict(torch.load('vgg16_cifar10.pth'))
```
Comment (chicm-ms) [inline examples/model_compress/MeanActivation_torch_cifar10.py]: It seems the `load_state_dict` and the retrain part should swap order, current logic is strange.

Comment (Cjkkkk) [reply examples/model_compress/MeanActivation_torch_cifar10.py]: I don't understand why is the logic strange. Could you be more specific?

Comment (chicm-ms) [reply examples/model_compress/MeanActivation_torch_cifar10.py]: if args.retrain is True,  we do not need to load model here.
if args.retrain is False, we  got a model with random parameter before `load_state_dict`,  our pruner should not prune a random model, so we should assert the 'vgg16_cifar10.pth' must exist.

Comment (Cjkkkk) [reply examples/model_compress/MeanActivation_torch_cifar10.py]: fixed

File: examples/model_compress/DoReFaQuantizer_torch_mnist.py (hunk 1 above)
Comment (Cjkkkk) [reply examples/model_compress/DoReFaQuantizer_torch_mnist.py]: fixed

**Pull request merged**